### DISCUSSION TITLE
Input quantity

### DISCUSSION THREAD

**Discussion by RobertCrx** [bootscore/bootscore#567] [Q&A] [answered]
How do I set input quantity to the products loop in shop loop and in the single product same as in the cart page ? Thanks

**Comment by crftwrk** [comment, marked as answer]
`functions.php`:

```php
/**
 * Show quantities next to add to cart buttons in loop
 */
add_filter( 'woocommerce_loop_add_to_cart_link', 'quantity_inputs_for_woocommerce_loop_add_to_cart_link', 10, 2 );
function quantity_inputs_for_woocommerce_loop_add_to_cart_link( $html, $product ) {
  if ( $product && $product->is_type( 'simple' ) && $product->is_purchasable() && $product->is_in_stock() && ! $product->is_sold_individually() ) {
    $html = wc_get_template_html( 'single-product/add-to-cart/simple.php' );
  }
  return $html;
}
```

You need of course some additional CSS to fix the layout.

Solved?

**Comment by RobertCrx** [comment]
Yes , but I would need to edit , because I want event to trigger on + or - or even input change , not on adding to cart , also I need to calculate and show the x amount of quantity before price when he change quantity

**Reply by crftwrk** [reply]
This goes beyound the scope of the project, you must custom code it by yourself.
